empress/__init__.py

```diff
@@ -107,17 +108,18 @@ def __init__(self, recongraph: dict, roots: list, n_recon: int, recon_input: Rec
         self.total_cost = total_cost
         self.n_recon = n_recon
         self.roots = roots
+        self.event_scores = self.__compute_event_frequencies()
```

Comment:
It seems that the code implicitly recompute event scores when you cluster, which is not the expected behavior.

Reply:
What you could do is to add `event_scores` as a required argument to `__init__` on line 102. Then, we could modify `_compute_event_frequencies` to be a static method which takes `ReconInput` and `recon_dict: dict`. We only call `_compute_event_frequencies` when we call `reconcile`. At other points, we copy the event scores down to lower level.

Reply:
changed to use set_event_frequencies in reconcile